dist/styles/_search.scss

```diff
@@ -51,3 +51,12 @@
         color: $input-color;
     }
 }
+
```

Comment:
I think you don't need `.blue-app-sidebar` around anymore when `<Search />` already has class name `.blue-app-search.blue-app-search-sidebar`.